engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -1135,92 +1135,105 @@ namespace dmGameSystem
             dmGui::TextureSetAnimDesc* anim_desc = dmGui::GetNodeTextureSet(scene, node);
             dmGameSystemDDF::TextureSet* texture_set_ddf = anim_desc ? (dmGameSystemDDF::TextureSet*)anim_desc->m_TextureSet : 0;
             bool use_geometries = texture_set_ddf && texture_set_ddf->m_Geometries.m_Count > 0;
+            const float* tc = dmGui::GetNodeFlipbookAnimUV(scene, node);
```

Comment:
We prefer early-outs in our code base.
If it's 0, do the untextured case, and continue. As we did before (although it could have been at the top)

Reply:
Answered: https://github.com/defold/defold/pull/4921#issuecomment-644750126